ymer/src/main/java/com/avanza/ymer/util/GigaSpacesInstanceIdUtil.java

```diff
@@ -35,9 +35,6 @@
 	private GigaSpacesInstanceIdUtil() {
 	}
 
-	/**
```

Comment:
isn't this documentation reference still interesting? Maybe just add a comment that the logic is the same as gigaspaces partitioning logic?

Reply:
I removed the reference because the referenced method no longer exists, so the javadoc plugin produced an error.
I have added javadoc including a couple of references in https://github.com/AvanzaBank/ymer/pull/77/commits/fbd3279a2494975b27a976173825719ee657fb12